import-scripts/verify-management-state.sh

```diff
@@ -10,42 +10,79 @@
 # - capture the current database color according to the management database table
 # - exit with error if they differ
 
-unset BLUE_SERVICE_LIST
-unset GREEN_SERVICE_LIST
-declare -a BLUE_SERVICE_LIST
-declare -a GREEN_SERVICE_LIST
-BLUE_SERVICE_LIST+=('cbioportal-backend-public-blue')
-BLUE_SERVICE_LIST+=('cbioportal-backend-public-beta-blue')
-BLUE_SERVICE_LIST+=('cbioportal-backend-master-blue')
-BLUE_SERVICE_LIST+=('cbioportal-backend-clickhouse-only-db-blue')
-GREEN_SERVICE_LIST+=('cbioportal-backend-public-green')
-GREEN_SERVICE_LIST+=('cbioportal-backend-public-beta-green')
-GREEN_SERVICE_LIST+=('cbioportal-backend-master-green')
-GREEN_SERVICE_LIST+=('cbioportal-backend-clickhouse-only-db-green')
-PUBLIC_CLUSTER_INGRESS_NAME='cbioportal-ingress'
-tmp="/data/portal-cron/tmp/import-cron-public"
+declare -a BLUE_SERVICE_LIST=()
+declare -a GREEN_SERVICE_LIST=()
+YQ_BINARY=${YQ_BINARY:-yq}
```

Comment:
If YQ_BINARY must be defined in the environment in order for this script to run successfully, we should (on beginning of execution) verify that YQ_BINARY is non-empty and if it is not we should exit with an error informing the user that they should source automation_environment.sh before attempting to run this script.